tests/DuskTestCase.php

```diff
@@ -41,4 +42,42 @@ protected function driver()
             )
         );
     }
+
+    public $passwordDefault = 'lifull@123';
```

Comment:
This password and password default in UserTableTestAddUserSeeder.php is same.
Why should define it as constant and get it in same place?

Reply:
I fixed, I add const passwordDefault in model